modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc

```diff
@@ -1,14 +1,14 @@
-= Debugging The Runtime Manager Agent
+= Troubleshooting the Runtime Manager Agent
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: agent, mule, servers, monitor, notifications, external systems, third party, get status, metrics
 
-In order to troubleshoot and provide better insights upon a malfunction of the Runtime Manager Agent installation, a new `$MULE_HOME/bin/amc_debug` script is available.
+A `$MULE_HOME/bin/amc_debug` script is available to help you troubleshoot Runtime Manager Agent installation issues.
 
-If you experience any issue with the Runtime Manager Agent Plugin and/or with managing applications on the Anypoint platform, it's highly recommended to obtain the output of this script before filling any support tickets.
+If you experience issues with the Runtime Manager Agent Plugin or with managing applications on Anypoint Platform, you should run and obtain the output of this script before filing a support ticket.
```

Comment:
let's not use "should" here? Should it be "...Anypoint Platform, run this script and save the output before...." (because support will want to see the output?)